Create special_case_issue.md issue template (#16452)
nvaccess/nvda: projectDocs/issues/githubIssueTemplateExplanationAndExamples.md

If you are not familiar, please take some time to learn about 
[Github markdown](https://guides.github.com/features/mastering-markdown/)

**Warning**: In all but exceptional circumstances we require one of these templates to be completed.

> Your issue will likely be closed if a template has not been followed.

Author: Ignore this rule in future

> We currently have four templates (click to jump to the section):

Author: ```suggestion
We currently have four templates:

```

If you are not familiar, please take some time to learn about 
[Github markdown](https://guides.github.com/features/mastering-markdown/)

**Warning**: In all but exceptional circumstances we require one of these templates to be completed.
Your issue will likely be closed if a template has not been followed.

We currently have four templates:

- For [feature requests](#feature-request-template)
- For [bug reports](#bug-report-template)
- For [special case issues](#special-case-issue-template) that cannot easily be categorised as either a bug report or a feature request
- For security vulnerabilities (please note that these are reported differently, for more information see https://github.com/nvaccess/nvda/blob/master/security.md)

## General information

> The following information applies to all issues (bug reports, new features, even pull requests).

Author: ```suggestion
The following information applies to all issues and pull requests.

```

**Warning**: In all but exceptional circumstances we require this template to be completed.
Your issue will likely be closed if this template has not been followed.

We currently have four templates:
- for bug reports
- for feature requests

> - for special case issues that cannot be easily categorised as either a bug report or a feature request

Reviewer: the rest of this document outlines how to fill out each of these templates in more detail. It is probably worth adding content for the new issue template. Historically we have tried to minimize the use of inline comments to explain fields in favour of putting the information in this document instead.

> - for security vulnerabilities

Reviewer: I would avoid conflating issues with advisories as they are reported very differently. I would at the very least note the difference and link to this: https://github.com/nvaccess/nvda/blob/master/security.md

functionality.
Thus, it is recommended to run the fixing tool whenever focus problems, performance problems on
websites or navigation problems in focus or browse mode on different interfaces are encountered.

## Special Case Issue template

### Detailed description of the issue
Provide a detailed description of the issue you are reporting. 
Include specific details to help us understand the context.

> ### Why are the other templates not appropriate in this case?

Author: Ignore this rule in future

Explain why this issue cannot be addressed using the standard bug report or feature request templates.

### Have you asked for advice on how to report this issue via a community discussion? If so, please link to the discussion
Mention if you have sought advice or shared this issue in community discussions before creating this issue ticket. 
Include links to discussion threads if applicable, for example:
- https://github.com/nvaccess/nvda/discussions
- https://nvda.groups.io/g/nvda

> ### Steps to reproduce or illustrate the issue (if applicable)

Reviewer: This line already is surrounded by blank lines. The suggested change applies no change

If possible, provide steps to demonstrate or reproduce the issue. 
If this or the following sections are not applicable, write "N/A."

### Expected outcome or behaviour (if applicable)
Describe what you expected to happen or how you expect the project to handle this kind of issue.

### Screenshots, logs, and other attachments (if applicable)

> Attach any relevant screenshots, logs, or files that would help in diagnosing or understanding the issue.

Reviewer 2: ```suggestion
Attach any relevant logs or files that would help in diagnosing or understanding the issue.
Since many contributors are blind, screenshots should preferably be provided only in case the issue is purely graphical, and an alternative textual description is welcome.
```

websites or navigation problems in focus or browse mode on different interfaces are encountered.

## Special Case Issue template

### Detailed description of the issue
Provide a detailed description of the issue you are reporting. 
Include specific details to help us understand the context.

### Why are the other templates not appropriate in this case?
Explain why this issue cannot be addressed using the standard bug report or feature request templates.

### Have you asked for advice on how to report this issue via a community discussion? If so, please link to the discussion
Mention if you have sought advice or shared this issue in community discussions before creating this issue ticket. 
Include links to discussion threads if applicable, for example:
- https://github.com/nvaccess/nvda/discussions
- https://nvda.groups.io/g/nvda

### Steps to reproduce or illustrate the issue (if applicable)
If possible, provide steps to demonstrate or reproduce the issue. 
If this or the following sections are not applicable, write "N/A."

### Expected outcome or behaviour (if applicable)
Describe what you expected to happen or how you expect the project to handle this kind of issue.

> ### Screenshots, logs, and other attachments (if applicable)

Author: ```suggestion
### Logs, screenshots and other attachments (if applicable)

```

functionality.
Thus, it is recommended to run the fixing tool whenever focus problems, performance problems on
websites or navigation problems in focus or browse mode on different interfaces are encountered.

## Special Case Issue template

### Detailed description of the issue
Provide a detailed description of the issue you are reporting.
Include specific details to help us understand the context.

### Why are the other templates not appropriate in this case?
Explain why this issue cannot be addressed using the standard bug report or feature request templates.

### Have you asked for advice on how to report this issue via a community discussion? If so, please link to the discussion
Mention if you have sought advice or shared this issue in community discussions before creating this issue ticket.

> Include links to discussion threads if applicable, for example:

Author: Double check your suggestions. Nothing has changed in your suggested change from what is there already.

- https://github.com/nvaccess/nvda/discussions
- https://nvda.groups.io/g/nvda

### Steps to reproduce or illustrate the issue (if applicable)
If possible, provide steps to demonstrate or reproduce the issue. 
If this or the following sections are not applicable, write "N/A."

### Expected outcome or behaviour (if applicable)
Describe what you expected to happen or how you expect the project to handle this kind of issue.

### Logs, screenshots and other attachments (if applicable)

Attach any relevant logs or files that would help in diagnosing or understanding the issue.

> Since many contributors are blind, screenshots should preferably be provided only in case the issue is purely graphical, and an alternative textual description is welcome.

Reviewer: I would suggest using the same content as the other template for this.
It might be worth doing the same for other sections that are duplicated.

```md
Refer to [Attachments / Images](#attachments--images).
```
